src/cheri/insns/auipc_32bit.adoc

```diff
@@ -19,7 +19,7 @@ include::base_isa_extension.adoc[]
 Description::
 Form a 32-bit offset from the 20-bit immediate filling the lowest bits with zero;
 the number of places to shift is determined by the capability encoding's choice of
-the <<section_auipc_shift>> value (12, unless otherwise specified).
+the <<section_auipc_shift>> value (12, unless otherwise specified by the capability encoding format).
```

Comment:
As I noted in the parametric base proposal, the AUIPC shift isn't _really_ a property of the encoding format, it's just that some encoding formats struggle with too large an AUIPC shift (but software on a smaller-mantissa system _could_ always replace a potentially-bounds-violating `AUIPC n; LY` with an always-in-bounds `AUIPC (n±k); ADDI; ...; ADDI; LY` to compensate!).  The AUIPC shift really is _just_ a parameter of the base.

Reply:
The ambition behind the parametric base proposal, which Alex has so kindly taken on and transmogrified in this PR, was to fully define the observable behavior of the capability getters and setters, leaving only the field layout (and so, necessarily, the behavior of some load and store instructions) to the encoding.  In this spirit, `AUIPC` is a capability setter, and its behavior is fully determined by the capability mantissa width and the shift applied to the immediate.

That is, I think the AUIPC shift belongs as a parameter in the `_Ly{...}` string, and I was just noting that, in contrast to the others, it is not as overtly tied to the encoding.

Reply:
I was thinking yesterday that we could actually omit the AS parameter and say that it's implied if representability isn't sufficient for +-2KB? I'll do more work on the PR later today and clarify more things such as adding the known parameters to the RVY chapter.